prow/gangway/gangway.proto

```diff
@@ -36,7 +36,17 @@ service Prow {
       get: "/v1/executions"
     };
   }
-}
+  rpc BulkJobStatusChange(BulkJobStatusChangeRequest) returns (JobExecutions) {
+    option (google.api.http) = {
+      custom: {
+        kind: "POST",
+        path: "/v1/status-change",
```

Comment:
`status-change` may be to vague. I would do something like `bulk-job-status-update`